recipes/tensorflow-lite/all/conanfile.py

```diff
@@ -32,26 +33,18 @@ class TensorflowLiteConan(ConanFile):
         "with_mmap": True,
         "with_xnnpack": True,
     }
-    generators = "cmake", "cmake_find_package"
-    exports_sources = ["CMakeLists.txt", "patches/**"]
 
-    _cmake = None
 
-    @property
-    def _module_subfolder(self):
-        return os.path.join("lib", "cmake")
-
-    @property
-    def _module_file(self):
-        return "conan-official-{}-targets.cmake".format(self.name)
+    short_paths = True
+    generators = "cmake", "cmake_find_package", "cmake_find_package_multi"
 
     @property
     def _source_subfolder(self):
-        return "source_subfolder"
+        return "src"
 
     @property
     def _build_subfolder(self):
-        return "build_subfolder"
+        return "build"
```

Comment:
Why do you need this change?

Reply:
Yes. Short paths works, but there are some limitation already commented by SpaceIm. Using src and build its an option on Windows. It's an exception, not the usual case.

Reply:
I still don't see the point of changing it, if it works with short paths